pkg/web/apps/apps_psql_repository.go

```diff
@@ -128,5 +130,154 @@ func (a *appsPostgreSQLRepository) GetAppByID(ID string) (*models.App, error) {
 	}
 
 	return &app, nil
+}
+
+// GetDeviceByDeviceID gets a device by its Device ID
+func (a *appsPostgreSQLRepository) GetDeviceByDeviceID(deviceID string) (*models.Device, error) {
```

Comment:
Is this function currently used anywhere?

Reply:
Doesn't seem to be. Dunno why it is here :grinning: